tools/githooks/pre-commit

```diff
@@ -0,0 +1,30 @@
+#================================================================
+# HEADER
+#================================================================
+#% DESCRIPTION
+#%    This is a git hook containing some commands to help you
+#%    automatically format the code. This will help to maintain
+#%    good quality code in the github repository. This script
+#%    will be executed before every commit you make.
+#% SET UP
+#%    OPTION 1:
+#%        Copy this file to the local folder .git/hooks of your
+#%        git repository.
+#%    OPTION 2:
```

Comment:
remove options and make option2 the default setup

Reply:
Done